### ISSUE TITLE
1.8.1 doesnt seem to work for 1.19.3

### ISSUE THREAD

**Issue by GoldenShad0w** [GamerCoder215/MobChip#52]
### Bug Type

Console Errors

### MobChip Version

1.8.1

### Server Version

Paper 1.19.3-442

### Description

Creating an EntityBrain object (`EntityBrain brain = BukkitBrain.getBrain((Mob) entity);`) throws an internal error

### Expected Behavior

Create the object

### Actual Behavior

```
[19:12:04 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'aurum' in plugin Aurum v1.0-SNAPSHOT
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R2.CraftServer.dispatchCommand(CraftServer.java:911) ~[paper-1.19.3.jar:git-Paper-442]
	at org.bukkit.craftbukkit.v1_19_R2.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64) ~[paper-1.19.3.jar:git-Paper-442]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[paper-1.19.3.jar:?]
	at net.minecraft.commands.Commands.performCommand(Commands.java:316) ~[?:?]
	at net.minecraft.commands.Commands.performCommand(Commands.java:300) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.performChatCommand(ServerGamePacketListenerImpl.java:2288) ~[?:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$20(ServerGamePacketListenerImpl.java:2248) ~[?:?]
	at net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:59) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.3.jar:git-Paper-442]
	at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
	at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1341) ~[paper-1.19.3.jar:git-Paper-442]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.19.3.jar:git-Paper-442]
	at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
	at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1318) ~[paper-1.19.3.jar:git-Paper-442]
	at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1311) ~[paper-1.19.3.jar:git-Paper-442]
	at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
	at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1289) ~[paper-1.19.3.jar:git-Paper-442]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1177) ~[paper-1.19.3.jar:git-Paper-442]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.19.3.jar:git-Paper-442]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError
	at goldenshadow.aurum.entities.CustomEntity.deserialize(CustomEntity.java:50) ~[aurum-1.0-SNAPSHOT.jar:?]
	at goldenshadow.aurum.Commands.onCommand(Commands.java:1689) ~[aurum-1.0-SNAPSHOT.jar:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	... 23 more
Caused by: java.lang.IllegalStateException: Invalid Version: 1_19_R2
	at me.gamercoder215.mobchip.abstraction.ChipUtil.getWrapper(ChipUtil.java:189) ~[aurum-1.0-SNAPSHOT.jar:?]
	at me.gamercoder215.mobchip.bukkit.BukkitBrain.<clinit>(BukkitBrain.java:43) ~[aurum-1.0-SNAPSHOT.jar:?]
	at goldenshadow.aurum.entities.CustomEntity.deserialize(CustomEntity.java:50) ~[aurum-1.0-SNAPSHOT.jar:?]
	at goldenshadow.aurum.Commands.onCommand(Commands.java:1689) ~[aurum-1.0-SNAPSHOT.jar:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	... 23 more
```

**Comment by SananR** [comment]
Can confirm that I am getting the same issue with version 1.8.1 and the Paper 1.19.3

**Comment by GamerCoder215** [comment]
Seems like an issue with reflection. I'll work on a fix available for v1.8.1.

**Comment by GamerCoder215** [comment]
Can you try running it on Spigot 1.19.3 instead of Paper 1.19.3? I thought it was an issue with finding the abstraction class, but it might be an issue with changes to Paper NMS.

**Comment by GoldenShad0w** [comment]
```
[11:50:16] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'aurum' in plugin Aurum v1.0-SNAPSHOT
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R2.CraftServer.dispatchCommand(CraftServer.java:847) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at org.bukkit.craftbukkit.v1_19_R2.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:50) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:306) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:290) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1957) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.network.PlayerConnection.lambda$18(PlayerConnection.java:1919) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.util.thread.IAsyncTaskHandler.b(SourceFile:67) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.util.thread.IAsyncTaskHandler.d(SourceFile:156) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(SourceFile:23) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1154) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:1) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.util.thread.IAsyncTaskHandler.x(SourceFile:130) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.MinecraftServer.bh(MinecraftServer.java:1133) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1126) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:139) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.MinecraftServer.i_(MinecraftServer.java:1110) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1021) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:301) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3652-Spigot-19641c7-6aeb5e4]
	at java.lang.Thread.run(Thread.java:1589) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError
	at goldenshadow.aurum.entities.CustomEntity.deserialize(CustomEntity.java:50) ~[?:?]
	at goldenshadow.aurum.Commands.onCommand(Commands.java:1816) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	... 23 more
Caused by: java.lang.IllegalStateException: Invalid Version: 1_19_R2
	at me.gamercoder215.mobchip.abstraction.ChipUtil.getWrapper(ChipUtil.java:189) ~[?:?]
	at me.gamercoder215.mobchip.bukkit.BukkitBrain.<clinit>(BukkitBrain.java:43) ~[?:?]
	at goldenshadow.aurum.entities.CustomEntity.deserialize(CustomEntity.java:50) ~[?:?]
	at goldenshadow.aurum.Commands.onCommand(Commands.java:1816) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.19.3-R0.1-SNAPSHOT.jar:?]
	... 23 more
```
Still get the same error with spigot

**Comment by GamerCoder215** [comment]
Fixed in [`47db82c`](https://github.com/GamerCoder215/MobChip/commit/47db82cc74c961d948e1f7358491b734e5d972f3). 

Updated v1.8.1 tag to reflect new changes, please re-download from CodeMC or JitPack ([download the new JAR](https://github.com/GamerCoder215/MobChip/releases/tag/1.8.1), build with the `-U` flag in Maven, or the `--refresh-dependencies` flag in Gradle)

Marking as closed.
